components/Layout/index.tsx

```diff
@@ -22,6 +22,21 @@ export const Layout: React.VFC<{
     <>
       <HeadDoc />
       <SCLabsTestHeader />
+
+      <nav>
+        <ul id="applicationTitle" className="z-10">
```

Comment:
this id should be removed or changed, as it will conflict with the `applicationTitle` element [here](https://github.com/DTS-STN/eligibility-estimator/blob/develop/components/Layout/index.tsx#L56)